source/libsass.html.haml

```diff
@@ -51,7 +51,7 @@ title: libSass
   %li#javascript
     :markdown
       ### JavaScript
-      The only pure JavaScript implementation is [sass.js](https://github.com/medialize/sass.js). There's a [browser-based way to test it out](http://medialize.github.io/sass.js/), too.
+      The only pure JavaScript implementation is [sass.js](https://github.com/medialize/sass.js). There's a [browser-based way to test it out](http://medialize.github.io/playground.sass.js/), too.
```

Comment:
Hey @rodneyrehm  — do you mind fixing the houndci error in this and then I'll merge it in? Just make sure the code is around 78-ish characters.